gemato/hash.py

```diff
@@ -28,6 +28,19 @@ def update(self, data):
 	def hexdigest(self):
 		return self.size
 
+class NoneHash(object):
```

Comment:
I don't really understand the purpose of this. You should be able to just use empty `hashes` list. If it doesn't work, let's fix that instead of adding verbose workaround for it.

Reply:
Ah, that is even better, will drop this commit.